core-api/src/main/java/com/hevelian/identity/core/api/validation/constraints/Logo.java

```diff
@@ -20,7 +20,7 @@
 @Retention(RUNTIME)
 public @interface Logo {
 
-  String message() default "Image width and height should not exceed 300 pixels";
+  String message() default "Invalid image file! Please, check upon the next requirements: file type is 'image/png' or 'image/jpeg'; width and height not exceed 300 pixels.";
```

Comment:
Too complex description. "Logo is not an 'image/png' or 'image/jpeg' file or width/height is greater than 300 pixels."

Reply:
This needs to be revised.